### DISCUSSION TITLE
IGNITE MST REACT NATIVE

### DISCUSSION THREAD

**Discussion by HAFDIAHMED** [mobxjs/mobx-state-tree#1808] [Q&A] [answered]
is there any one has a free tutorial to make a react native using ignite boilerplate and MST ( MOBX STATE TREE)

**Comment by EmilTholin** [comment, marked as answer]
Hi @HAFDIAHMED!

Robin Heinze, who works at Infinite Red, has a nice [tutorial on how to build a Trivia app with Ignite](https://shift.infinite.red/creating-a-trivia-app-with-ignite-bowser-part-1-1987cc6e93a1).

**Reply by HAFDIAHMED** [reply]
Thank you so much@EmilTholin
